theme-shop/download.py

```diff
@@ -0,0 +1,330 @@
+import pycurl, os, sys, binascii, hashlib, struct
+from io import BytesIO, SEEK_CUR
+from boss import BOSSFile
+from pathlib import Path
+from pycurl import Curl
+
+CTR_PEM_PATH = os.environ["CTR_PROD_3"]
+OUTPUT_PATH = os.curdir
+DL_MAX_CONN = 100
+
+COUNTRIES = [
+	'JP', 'AI', 'AG', 'AR', 'AW', 'BS', 'BB', 'BZ',
+	'BO', 'BR', 'VG', 'CA', 'KY', 'CL', 'CO', 'CR',
+	'DM', 'DO', 'EC', 'SV', 'GF', 'GD', 'GP', 'GT',
+	'GY', 'HT', 'HN', 'JM', 'MQ', 'MX', 'MS', 'AN',
+	'NI', 'PA', 'PY', 'PE', 'KN', 'LC', 'VC', 'SR',
+	'TT', 'TC', 'US', 'UY', 'VI', 'VE', 'AL', 'AU',
+	'AT', 'BE', 'BA', 'BW', 'BG', 'HR', 'CY', 'CZ',
+	'DK', 'EE', 'FI', 'FR', 'DE', 'GR', 'HU', 'IS',
+	'IE', 'IT', 'LV', 'LS', 'LI', 'LT', 'LU', 'MK',
+	'MT', 'ME', 'MZ', 'NA', 'NL', 'NZ', 'NO', 'PL',
+	'PT', 'RO', 'RU', 'RS', 'SK', 'SI', 'ZA', 'ES',
+	'SZ', 'SE', 'CH', 'TR', 'GB', 'ZM', 'ZW', 'AZ',
+	'MR', 'ML', 'NE', 'TD', 'SD', 'ER', 'DJ', 'SO',
+	'AD', 'GI', 'GG', 'IM', 'JE', 'MC', 'TW', 'KR',
+	'HK', 'MO', 'ID', 'SG', 'TH', 'PH', 'MY', 'CN',
+	'AE', 'IN', 'EG', 'OM', 'QA', 'KW', 'SA', 'SY',
+	'BH', 'JO', 'SM', 'VA', 'BM'
+]
+
+LANGUAGES = [ 'ja', 'en', 'fr', 'de', 'it', 'es', 'zh', 'ko', 'nl', 'pt', 'ru', 'zh_trad' ]
```

Comment:
Where did `zh_trad` come from? I've never seen this anywhwere before

Reply:
I'm told that came from REing the eshop module

Reply:
Does it actually work though? If so, it should probably be added to the SpotPass archiver too

Reply:
That's fair. It can stay as-is